lib/generators/ember/bootstrap_generator.rb

```diff
@@ -68,6 +60,17 @@ def inject_into_application_file(safe_extension)
           template application_file, full_path
         end
       end
+
+      def get_options_from_path(path)
```

Comment:
Yes, both of them.
It seems this method takes as a file contents as an argument, not a file path.